INSTALLATION_MACOS.md

```diff
@@ -0,0 +1,70 @@
+# Installation
+
+## Install Homebrew, package manager for MacOS
+[Homebrew][brew]
+
+```bash
+xcode-select --install
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+```
+
+## Install ARM cross-compiler
+[homebrew-arm][arm-cross]
+
+```bash
+brew tap osx-cross/arm
+brew install arm-gcc-bin
+```
+
+## Install lm4tools
+[lm4tools][lm4tools-link]
+
+```bash
+cd lm4tools
+brew install libusb
+
+# In a directory of your choice
+git clone https://github.com/utzig/lm4tools.git
+cd lm4tools/lm4flash
+make
+
+# Update your path
+echo 'PATH=$PATH:'$PWD >> ~/.bashrc
+```
+
+> NOTE: If you use ZSH, then update the append to the ~/.zshrc file instead of ~/.bashrc
+
+## Install OpenOCD
+[OpenOCD][openocd-link]
```

Comment:
I just added a missing "o" here, but everything looks great! Thanks for putting these detailed install instructions together. You might want to add a link to this file in the README so people know to look for it when they're getting started! :)